src/test/java/org/commcare/formplayer/tests/BaseTestClass.java

```diff
@@ -946,6 +955,16 @@ private <T> T generateMockQuery(ControllerType controllerType,
                 break;
         }
         switch (requestType) {
+            case MULTIPART:
+                MockPart answer = new MockPart(PART_ANSWER, ((String)bean).getBytes());
+                answer.getHeaders().setContentType(MediaType.APPLICATION_JSON);
+                result = controller.perform(
+                        multipart(urlPrepend(urlPath))
+                                .file(file)
+                                .part(answer)
+                                .cookie(new Cookie(Constants.POSTGRES_DJANGO_SESSION_ID, "derp")));
+                break;
```

Comment:
Is this change still necessary?

Reply:
https://github.com/dimagi/formplayer/pull/1198/commits/324daa004a2120df5492e165150960b84c953ecf